### PR TITLE
Culture 

### PR DESCRIPTION
**Pull request by ThatOneGuy42**
I FUCKED UP AT LEAST 3 TIMES TRYING TO DO THIS BUT I DID IT RIGHT THIS TIME I HATE MYSELF

Anyways this adds Culture Folder, The Arts Page is moved from DYK to it's own thing, and adds the Religion page. Now the Culture folder can be filled with a variety of things.

### CODE REVIEW

File: Culture/Religion.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## The Old World Faiths
+
+ It can generally assumed that most, if not all of the major faiths of 20th Century Humanity have survived the last several hundred 
+years into the current day, especially so with Christianity, Islam, Hinduism and Buddhism who were by far the largest religions of the 
+time. Many other old-age religions of human history have survived through the centuries but not to the same degree as the Big Four. 
+
+### Christianity
+	
+	Christianity exists much in the same way that it did long ago- Splintered all around by the various different ideas and faiths that 
+make it up from Catholicism to Orthodox Christianity to Mormonism, and even the new-age beliefs such as Joshuaism and Heterdox
```
Comment (babydoll) [inline Culture/Religion.md]: heterodox

**Comment by Raptor1628** [comment]
Dude, how many commits do you need? Do you just want me to do it? :P

**Comment by ThatOneGuy42** [comment]
I MESSED UP AND IT WOULDN'T GET RID OF THE OTHER COMMENTS DON'T BE MAD.
And no its alright, after this I have another PR going in.

Also yes, heterodoxy, a extremely hardcore, reactionary branch of the
faith. Aliens are soulless beings not the children of God but adversaries,
cloning is wrong, robots are wrong, colonization is wrong cause all we need
is earth, etc. That sorta rjimf.

**Comment by babydoll** [comment]
No, you spelled Heterodox wrong

**Comment by ThatOneGuy42** [comment]
I fixed it.

**Comment by Raptor1628** [comment]
This uh..isn't fixed :P

**Comment by ThatOneGuy42** [comment]
I am terribad with git and no sure how fix it plox help

**Comment by ThatOneGuy42** [comment]
Im fixing soon

**Pull request closed without merging**